src/main.cpp

```diff
@@ -70,9 +73,28 @@ static inline void createGlobals() {
     arrowHandler = std::make_unique<ArrowHandler>(*escapeCodeGenerator);
 }
 
+static auto readConfiguration() {
+    passwd *pw = getpwuid(getuid());
+    std::string const configFile = std::string{pw->pw_dir} + "/.yebash.cfg";
```

Comment:
I mean only that the `.yebash.cfg` name reminds me old Windows configuration files naming convention. I would rather choose `.yebashrc` as it's quite Unix-ish ;) Yes, I know I'm nitpicking ;)